packages/bindings/src/serialport_win.cpp

```diff
@@ -56,7 +56,7 @@ void AsyncCloseCallback(uv_handle_t* handle) {
   delete async;
 }
 
-void EIO_Open(napi_env env, void* req) {
+void EIO_Open(void* req) {
```

Comment:
env was never used, and I believe it should not be used within the Execute method of the AsyncWorker